### PR TITLE
Issue4

### PR DESCRIPTION
**Pull request by XeBoris**
I address #4 here: Patrick asked if it its possible to load a specific cax.json file into cax. I also changed the Stockholm raw data path in cax.json.

### CODE REVIEW

**Comment by JelleAalbers** [comment]
Thanks! Could you maybe remove the various `cax.egg-info` and `cax.log` files, or better yet add them to the `.gitignore`?

**Comment by XeBoris** [comment]
Should be done now.
Sorry, my first commit :)

Cheers,
Boris

On 03.05.2016 13:56, Jelle Aalbers wrote:
> Thanks! Could you maybe remove the various `cax.egg-info` and `cax.log` files, or better yet add them to the `.gitignore`?
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/XENON1T/cax/pull/10#issuecomment-216506586
> 

-- 
Boris Bauermeister
Stockholms Universitet
Kosmologi astropartikelfysik och strängteori
106 91 Stockholm

**Comment by JelleAalbers** [comment]
No problem, it looks like they're still in there though.

**Comment by pdeperio** [comment]
It looks like this still requires the cax.json to be in the same directory as the original one, or am I reading the code wrong? 

I think we need to be able to specify the whole path of the file, since not all users will have write access to central installations of cax.

**Comment by XeBoris** [comment]
Ah! That was not my first intention. Not everybody should be allowed to
sync whatever he wants, shouldn't he?
Therefore I would suggest to have a pre-defined set of caxVERSION.json
files which match our needs here.
But if it is wished, I can do a further modification here.

Cheers,
Boris

On 03.05.2016 15:40, Patrick de Perio wrote:
> It looks like this still requires the cax.json to be in the same directory as the original one, or am I reading the code wrong? 
> 
> I think we need to be able to specify the whole path of the file, since not all users will have write access to central installations of cax.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/XENON1T/cax/pull/10#issuecomment-216530611
> 

-- 
Boris Bauermeister
Stockholms Universitet
Kosmologi astropartikelfysik och strängteori
106 91 Stockholm

File: cax/main.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def main():
       print('-----------------------------------------------------')
     set_json( caxjson_config )
 
-    
+    #exit()
```
Comment (tunnell) [inline cax/main.py]: remove?

File: cax/main.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def main():
       print('-----------------------------------------------------')
```
Comment (tunnell) [inline cax/main.py]: Why print and not log?

File: cax.log [hunk 3]
```diff
@@ -591,3 +591,12 @@ scp.SCPException: Error receiving, socket.timeout
 2016-05-03 09:35:49,199 [INFO] Executing ProcessBatchQueue.
```
Comment (tunnell) [inline cax.log]: `git rm cax.log`

**Comment by tunnell** [comment]
I don't get why you want to have versions of this?  Currently, the cax.json lives within the python package.  This gets updated with cax.  I don't get what this is trying to address.

Testing?

**Comment by pdeperio** [comment]
Yes, site independent testing and development

**Comment by tunnell** [comment]
@JelleAalbers  said he already put it in?

**Comment by JelleAalbers** [comment]
No, I only put in the option to specify command-line arguments.

**Comment by pdeperio** [comment]
cleaned up and test in https://github.com/XENON1T/cax/pull/18

**Pull request closed without merging**